pkg/doc/jose/common.go

```diff
@@ -8,46 +8,61 @@ package jose
 
 // IANA registered JOSE headers (https://tools.ietf.org/html/rfc7515#section-4.1)
 const (
-	// HeaderAlgorithm identifies the cryptographic algorithm used to secure the JWS.
+	// HeaderAlgorithm identifies...
```

Comment:
nit: why `...` instead of `:`

Reply:
I actually went back and forth between `...` and `:`. It was a toss-up for me, but if the `...` looks odd to you then I'm sure it'll look odd to others as well.

I've gone ahead and changed the `...` to `:` :)